spec/plaintime.html

```diff
@@ -516,9 +516,7 @@ <h1>
         1. Let _milliseconds_ be _ms2_ - _ms1_.
         1. Let _microseconds_ be _mus2_ - _mus1_.
         1. Let _nanoseconds_ be _ns2_ - _ns1_.
-        1. Let _norm_ be NormalizeTimeDuration(_hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_).
-        1. Assert: NormalizedTimeDurationAbs(_norm_).[[TotalNanoseconds]] &lt; nsPerDay.
```

Comment:
I'm not sure this is correct; `NormalizeTimeDuration` checks `abs(nanoseconds) ≤ maxTimeDuration`, and this is a check against `nsPerDay`, which is much smaller.

Reply:
Oops, you are totally right about this